src/evo/chainhelper.h

```diff
@@ -5,6 +5,8 @@
 #ifndef BITCOIN_EVO_CHAINHELPER_H
 #define BITCOIN_EVO_CHAINHELPER_H
 
+#include <uint256.h>
```

Comment:
nit: you could just
```
class uint256;
```

to avoid the include in the header?

Reply:
Resolved in latest push!